src/pages/Quiz/QuizPage.jsx

```diff
@@ -0,0 +1,128 @@
+import { useState } from "react";
+import PageLayout from "../../components/PageLayout";
+import Button from "@mui/material/Button";
+import LinearProgress from '@mui/material/LinearProgress';
+import { v4 as uuidv4 } from "uuid";
+import { Box } from "@mui/system";
+import { useTheme } from '@mui/material/styles';
+
+/**
+ * Renders the Assessment page.
+ * @returns {JSX.Element} QuizPage component
+ */
+
+function QuizPage() {
+
+    const theme = useTheme();
+    const [currentQuestion, setCurrentQuestion] = useState(0);
+    const [progressValue, setProgressValue] = useState(1);
+
+    const handleAnswerButtonClick = (answer) => {
+        if (dummyQuestions[currentQuestion].correct === answer.target.textContent) // checks if the answer is correct
+            console.log("Correct");
+        else
+            console.log("Incorrect");
```

Comment:
You consider to show that as visual representation to the user, but I think that is @NicolasKmbr decision